src/vs/workbench/api/node/extHostApiCommands.ts

```diff
@@ -239,6 +240,16 @@ export class ExtHostApiCommands {
 					{ name: 'column', description: '(optional) Column in which to open', constraint: v => v === void 0 || typeof v === 'number' }
 				]
 			});
+
+		this._register('vscode.quickOpen', (prefix: string, showOptions: IShowOptions) => {
```

Comment:
I am not sure about exposing `IShowOptions` to extensions, I would not do this until we hear otherwise.